docs/reference/components/sinks/clickhouse.cue

```diff
@@ -120,6 +120,12 @@ components: sinks: clickhouse: {
 				examples: ["mytable"]
 				syntax: "literal"
 			}
+		skip_unknown_fields: {
+			common:      true
+			description: "Sets `input_format_skip_unknown_fields`, allowing Clickhouse to discard fields not present in the table schema."
+			required:    false
+			type: bool: default: false
+		}
```

Comment:
Is this whitespace right? Probably needs a `cue fmt`

Reply:
🤷 that test is blowing up for a different reason locally - definitely forgot to check that